python-spec/src/somacore/scene.py

```diff
@@ -9,19 +9,21 @@
 from . import collection
 from . import data
 
-_DF = TypeVar("_DF", bound=data.DataFrame)
-"""A particular implementation of DataFrame."""
-_SpatialColl = TypeVar(
-    "_SpatialColl", bound=collection.Collection[Union[data.DataFrame, data.NDArray]]
-)
+_SpatialDF = TypeVar(
+    "_SpatialDF", bound=data.DataFrame
```

Comment:
@jp-dark, are we settled on that we will always need exactly one spatial DF per scene? or are there any cases in which we need to follow the same logic as _ImageColl being a collection?

I can't think of a case for multiple spatialDFs, so I'm ok with this. But it may be worth checking with @aaronwolen

Reply:
For the TileDB implementation, you may want to split up a spatial dataframe into multiple TileDB arrays. For example, the geospatial team does that for large point clouds using octrees for look-up. However, I think this type would still be a GeometryDataFrame or PointCloud type with extra additional functions for retrieving subsets of the data.

That being said, we should explore this question in more detail.